### ISSUE TITLE
British Virgin Islands +1 284 393 XXX validation issues

### ISSUE THREAD

**Issue by agri0026** [googlei18n/libphonenumber#1436]
We are trying to validate some phone numbers originating from British Virgin Islands using the +1 284 393 XXX extension and unfortunately the library is not recognizing the said numbers.

![gzejjer vergni](https://cloud.githubusercontent.com/assets/5612762/20397430/fccff104-ace9-11e6-8e9c-fb4bc6729837.jpg)

**Comment by penmetsaa** [comment]
@agri0026,
Thanks for providing the details! We've accepted this issue and
will fix this in an upcoming release.

Please note that you have submitted this information with the knowledge that we
may use, modify, reproduce, publicly display, and distribute the information in
accordance with our open source policies. Please let us know if you did not
intend to submit the information for such use.

**Comment by penmetsaa** [comment]
@agri0026 ,
Need one clarification. The range you mentioned +1 284 393 XXX is of 6 digits (excluding country prefix). But as per our evidences and online references it is 7 digit range.  
Ex: http://rhswcaribbean.com/contact/

The length of all mobile ranges in Virgin Islands is of 7 digits.
Is +1 284 393 XXX a typo?. If not please share any official evidence you may have. See our
[guidelines](https://github.com/googlei18n/libphonenumber/blob/master/CONTRIBUTING.md#provide-supporting-evidence).

As per existing evidences we are adding support for +1 284 393 XXXX (7 digit) mobile numbers.

**Comment by agri0026** [comment]
Yes I do confirm that I had an omission error. It should have read +1 284 393 XXXX. Please refer to the link for confirmation.

http://www.zolfocooper.com/contact-us

**Comment by penmetsaa** [comment]
Thanks for the confirmation :)
